javascript/como-javascript-funciona.md

```diff
@@ -0,0 +1,146 @@
+[Artigo original: https://blog.sessionstack.com/how-does-javascript-actually-work-part-1-b0bacc073cf](https://blog.sessionstack.com/how-does-javascript-actually-work-part-1-b0bacc073cf)
+
+---
+
+### Como Javascript funciona: uma visão geral da engine, do runtime e da pilha de chamadas
+
+Como JavaScript está ficando cada vez mais popular, times tem se aproveitado de seu suporte para vários papéis em suas funções - front-ned, back-end, aplicativos hibridos e muito mais.
```

Comment:
*front-end, híbridos tem acento e, por fim, acho que seria legal colocar trocar o primeiro - por :, por exemplo: "... papéis em suas funções : front-ned, back-end, aplicativos híbridos e muito mais."